### ISSUE TITLE
WMR 1.4.1 incompatibility with Rollup plugins using the `resolveId` hook

### ISSUE THREAD

**Issue by BenoitZugmeyer** [preactjs/wmr#490]
**Describe the bug**

I am trying to upgrade to WMR 1.4, and the `wmr build` breaks because of a small plugin using the `resolveId` hook I created to use absolute module paths. If it is expected, please point me to a better way to achieve this :) 

**To Reproduce**
See https://gist.github.com/BenoitZugmeyer/9feeccce91fc5e7aa2b71eb88bb4175f

The build fails with:

```
Invalid specifier: ~/App
    at /.../node_modules/wmr/wmr.cjs:2:1163619
    at /.../node_modules/wmr/wmr.cjs:2:1113671
    at Object.resolveId (/.../node_modules/wmr/wmr.cjs:2:1162593)
    at /.../node_modules/wmr/wmr.cjs:2:336874
```

**Expected behavior**
The build succeed.

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] macOS
 - Browser [e.g. chrome, safari] N/A
 - WMR Version [e.g. 22] 1.4.1

**Additional context**

This worked with 1.3.2. A `git bisect` indicates that the culprit might be https://github.com/preactjs/wmr/commit/0737249a5888e098c6fa65eb28e77f1ab966061a .

**Comment by developit** [comment]
@BenoitZugmeyer Thanks for the awesome bug report. The fix was just published in [1.4.2](https://github.com/preactjs/wmr/releases/tag/wmr%401.4.2).

**Comment by BenoitZugmeyer** [comment]
Thanks a lot for the quick fix, as always!
